GunShipAlpha/src/Enemy.java

```diff
@@ -38,4 +78,17 @@ public boolean isIntersectingLine(LineLogic line) {
     private Rectangle createBounds() {
         return new Rectangle(x, y, EnemySprite.width(), EnemySprite.height());
     }
+    public Projectile fireAtPlayer() {
+        double targetX = 400;
+        double targetY = 400;
+        double directionX = targetX - this.x;
+        double directionY = targetY - this.y;
+        double magnitude = Math.sqrt(directionX * directionX + directionY * directionY);
+        directionX /= magnitude;
+        directionY /= magnitude;
+        double offsetX = directionX * EnemySprite.width() / 2;
```

Comment:
magic number 2 is... not quite so bad? people mostly know it means half, although in gomarine i have stuff like 
EnemySprite.halfWidth and EnemySprite.halfHeight defined. Also... ask the enemy sprite to do something rather than just getting the width and height.